tsconfig.json

```diff
@@ -11,7 +11,7 @@
         "target": "es2019",
         "sourceMap": true,
         "outDir": "./dist",
-        "rootDir": "./",
+        "rootDir": "./src",
```

Comment:
The other thing that broke for me was having the `rootDir` be a parent of the `src` directory because it ends up creating an extra creating a `dist/src/` structure, which makes the references from the sub-`package.json` files awkward. I originally thought that this was completely necessary, but I think the final approach I took could have just replaced `dist` with `dist/src` in relative imports.

I decided to keep this anyway.